doc/ref_cert/README.md

```diff
@@ -2,8 +2,137 @@
 
 # CNTT Reference Conformance
 
-<a name="available-cr"></a>
+All CNTT conformance suites fulfill the following test case integration
+requirements to guarantee a smooth overall integration, the same end user
+actions and a unique test result format (e.g. OPNFV test result database)
+needed by the end users and the test case result verification programs (e.g.
+[OVP](https://www.opnfv.org/verification)). Historically, these rules were
+agreed by RC1 team and have been applied since
+[CNTT Snezka](https://github.com/cntt-n/CNTT/wiki/Snezka). The new RC2 stream
+asks to move them in this common place applicable to all CNTT conformance
+suites.
+
+<a name="ri-rc-toolchaings"></a>
+## CNTT RI and RC toolchains
+
+[OPNFV](https://www.opnfv.org/) has built a complete CI/CD toolchain
+continuously deploying and testing NFVI.
+
+As for all OPNFV installer projects,
+[Jenkins](https://build.opnfv.org/ci/view/cntt/) triggers scenario deployments,
+runs the OPNFV gating test cases and then publishes all
+test results in the
+[centralized test database](https://docs.opnfv.org/en/stable-hunter/_images/OPNFV_testing_working_group.png)
+and all artifacts (reports, logs, etc.) to
+[an S3 compatible storage service](http://artifacts.opnfv.org/).
+
+The CNTT verification and conformance processes will leverage existing OPNFV
+testing knowledge (projects) and experience (history) and then will conform
+to the overall toolchain design already in-place. The RC toolchain only
+requires for the local deployment of the components instead of leveraging
+the common OPNFV centralized services. But the interfaces remain unchanged
+mainly leveraging jenkins jobs, the common test case execution, the test
+result database and the S3 protocol to publish the artifacts. It's worth
+mentioning that dumping all results and logs required by conformance is already
+in place in CIRV (see
+[cntt-latest-zip](https://build.opnfv.org/ci/job/cntt-latest-zip/)) and
+Functest daily jobs (see
+[functest-hunter-zip](https://build.opnfv.org/ci/job/functest-hunter-zip/3/console))
+
+It should be noted that
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) supports both
+centralized and distributed deployment models as described below. It has
```

Comment:
"... deployment models as described below" but there are no such deployment models described in the text.

Reply:
in CNTT RI and RC toolchains. I could replace below if it helps. Updating